### DISCUSSION TITLE
cuda plotter v3 doesn't start in x64 windows

### DISCUSSION THREAD

**Discussion by RealStrike** [madMAx43v3r/chia-gigahorse#288] [General]
cuda_plot_k32_v3 doesn't start in x64 windows system. ) I have installed windows server, complains about incompatibility. This has never happened before, Is it just me? I didn't find anyone else who had problems with this, or did everyone switch to Linux?

**Comment by kvizena** [comment]
Not sure you can install a non-x64 Windows Server anymore, so not likely an issue.  I am running it fine on five machines with Nvidia cards, three workstations and two servers. 

Running PowerShell as an admin? 


From the README:

You might also have to install latest Microsoft Visual C++ Redistributable: 
https://aka.ms/vs/17/release/vc_redist.x64.exe
